test/controllers/helpers/htmlAssert.helper.ts

```diff
@@ -0,0 +1,47 @@
+import { JSDOM } from 'jsdom'
+
+import Optional from 'app/models/optional'
+
+export class HtmlAssertHelper {
```

Comment:
See `selectDirector.controller.test.ts` for usage

Enables granular testing on controller -> nunjucks integration, eliminating the need to run the app to test your template's behaviour

TODO - make default